modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ContextMenuContent.java

```diff
@@ -650,15 +650,19 @@ private void processRightKey(KeyEvent ke) {
 
     private void showMenu(Menu menu) {
         menu.show();
-        // request focus on the first item of the submenu after it is shown
-        ContextMenuContent cmContent = (ContextMenuContent)submenu.getSkin().getNode();
-        if (cmContent != null) {
-           if (cmContent.itemsContainer.getChildren().size() > 0) {
-               cmContent.itemsContainer.getChildren().get(0).requestFocus();
-               cmContent.currentFocusedIndex = 0;
-           } else {
-               cmContent.requestFocus();
-           }
+
+        // if there is a submenu
+        if (submenu != null) {
+            // request focus on the first item of the submenu after it is shown
+            ContextMenuContent cmContent = (ContextMenuContent)submenu.getSkin().getNode();
+            if (cmContent != null) {
```

Comment:
just a mini-note: personally, I prefer early-return on no-match (vs. wrapping nearly the whole method inside an if match

    if (submenu == null) return;

Reply:
I prefer a single point of return for small methods. Anyway, this file has lot of early returns. Hence, I am modifying as per your suggestion.